src/frontends/paddle/tests/test_models/gen_scripts/generate_generate_proposal_v2.py

```diff
@@ -60,15 +60,14 @@ def generate_proposals_v2(name: str, input_data: dict, attr: dict):
                 'anchors': anchors_np,
                 'var': variances_np
             },
-            fetch_list=[rois, roi_probs, rois_num],
-            return_numpy=False)
+            fetch_list=[rois, roi_probs, rois_num])
 
         # Save inputs in order of OpenVINO model, to facilite Fuzzy test, 
         # which accepts inputs and outputs in this order as well. 
-        saveModel(name, exe, feedkeys=['scores', 'bbox_deltas', 'im_shape', 'anchors', 'var'],
+        saveModel(name, exe, feed_vars=[scores, bbox_deltas, im_shape, anchors, variances],
                   fetchlist=[rois, roi_probs, rois_num],
                   inputs=[scores_np, bbox_deltas_np, im_shape_np, anchors_np, variances_np],
-                  outputs=outs, target_dir=sys.argv[1])
+                  outputs=[outs[0], outs[1], outs[2]], target_dir=sys.argv[1])
```

Comment:
Isn't it the same as 'outs'?

Reply:
yes, all other similar cases are all same as 'outs'. Do we need change it to 'outs'?

Reply:
yes

Reply:
> yes

done https://github.com/openvinotoolkit/openvino/pull/23010/commits/38c1e6d4be184c5c7522fe1a65f17b6b6f6bc2b6